dss-api.yml

```diff
@@ -56,13 +56,20 @@ basePath: /v1
 produces:
   - application/json
 securityDefinitions:
-  googAuth:
+  fusilladeAuth:
+    description: >
+      The following claims must be present in the JWT:
+      - 'https://auth.data.humancellatlas.org/group': the user group this user belongs must be hca or public
+      - 'iss': must be https://humancellatlas.auth0.com/ or a trusted google project
+      - 'aud': audience must be https://dev.data.humancellatlas.org/ or https://data.humancellatlas.org/
+      - 'email': the email of the requester,
     type: oauth2
```

Comment:
We should probably change this to `type: openIdConnect` unless there's a good reason not to.